apps/www/src/content/docs/installation/vite.md

```diff
@@ -27,62 +27,23 @@ npm create vite@latest my-vue-app -- --template vue-ts
 
 Install `tailwindcss` and its peer dependencies, then generate your `tailwind.config.js` and configure `postcss` plugins
```

Comment:
From what I understand they removed the config file.

Reply:
nvm me https://tailwindcss.com/docs/upgrade-guide#using-a-javascript-config-file